pyomo/solvers/tests/mip/test_mip.py

```diff
@@ -13,9 +13,34 @@
 
 import os
 from os.path import abspath, dirname
+
+from pyomo.solvers.plugins.solvers.CBCplugin import CBCSHELL
+
```

Comment:
Instead of hijacking the `test_mip` infrastructure, can you add these tests to `pyomo/solvers/tests/checks/test_CBCplugin:TestCBCUsingMock`?